packages/generator/src/edmx-to-vdm/common/enum-type.ts

```diff
@@ -18,7 +25,85 @@ export function transformEnumTypesBase(
     return {
       originalName: e.Name,
       typeName,
-      members: e.Member.map(m => m.Name)
+      underlyingType: e.UnderlyingType || 'Edm.Int32',
+      members: parseMember(e)
     };
   });
 }
+
+function parseMember(
+  edmxEnumType: EdmxEnumType
+): Record<string, number | BigNumber> {
+  validateUniqueness(edmxEnumType);
+  validateUnderlyingType(edmxEnumType);
+  edmxEnumType.UnderlyingType = edmxEnumType.UnderlyingType
+    ? edmxEnumType.UnderlyingType
+    : 'Edm.Int32';
+
+  if (!hasValidValues(edmxEnumType)) {
+    logger.warn(
+      `The enum ${edmxEnumType.Name} has invalid member values, which should be either of the following: 1. All values are specified. 2. No values are specified. 0 based index is used as value.`
+    );
```

Comment:
I think I would really only log this warning if `IsFlags` is not set to true. As we are not parsing this at the moment, I would just leave it, unless you think it is worth parsing it. I don't think it makes a difference for the SDK.